framework/messages/hr/yii.php

```diff
@@ -0,0 +1,105 @@
+    'The file "{file}" is too small. Its size cannot be smaller than {limit, number} {limit, plural, one{byte} other{bytes}}.' => 'Podatak "{file}" je premalen. Ne smije biti manji od {limit, number} {limit, plural, one{bajt} other{bajtova}}.',
+    'The format of {attribute} is invalid.' => 'Format od {attribute} je nevažeći.',
+    'The image "{file}" is too large. The height cannot be larger than {limit, number} {limit, plural, one{pixel} other{pixels}}.' => 'Slika "{file}" je prevelika. Visina slike ne smije biti veća od {limit, number} {limit, plural, one{piksel} other{piksela}}.',
+    'The image "{file}" is too large. The width cannot be larger than {limit, number} {limit, plural, one{pixel} other{pixels}}.' => 'Slika "{file}" je prevelika. Širina slike ne smije biti veća od {limit, number} {limit, plural, one{piksel} other{piksela}}.',
+    'The image "{file}" is too small. The height cannot be smaller than {limit, number} {limit, plural, one{pixel} other{pixels}}.' => 'Slika  "{file}" je premalena. Visina slike ne smije biti manja od {limit, number} {limit, plural, one{piksel} other{piksela}}.',
+    'The image "{file}" is too small. The width cannot be smaller than {limit, number} {limit, plural, one{pixel} other{pixels}}.' => 'Slika "{file}" je premalena.  Širina slike ne smije biti manja od {limit, number} {limit, plural, one{piksel} other{piksela}}.',
+    'The verification code is incorrect.' => 'Kod za provjeru nije točan.',
+    'Total <b>{count, number}</b> {count, plural, one{item} other{items}}.' => 'Ukupno <b>{count, number}</b> {count, plural, one{red} other{redova}}.',
+    'Unable to verify your data submission.' => 'Nije moguće provjeriti podnesene podatke.',
+    'Unknown command "{command}".' => 'Nepoznata naredba "{command}".',
+    'Unknown option: --{name}' => 'Nepoznata opcija: --{name}',
+    'Update' => 'Uredi',
+    'View' => 'Pregled',
+    'Yes' => 'Da',
+    'You are not allowed to perform this action.' => 'Nije vam dopušteno obavljati tu radnju.',
+    'You can upload at most {limit, number} {limit, plural, one{file} other{files}}.' => 'Najviše možete uploadat {limit, number} {limit, plural, one{fajl} other{fajlova}}.',
+    'in {delta, plural, =1{a day} other{# days}}' => 'u {delta, plural, =1{dan} other{# dana}}',
+    'in {delta, plural, =1{a minute} other{# minutes}}' => 'u {delta, plural, =1{minuta} other{# minuta}}',
+    'in {delta, plural, =1{a month} other{# months}}' => 'u {delta, plural, =1{mjesec} other{# mjeseci}}',
+    'in {delta, plural, =1{a second} other{# seconds}}' => 'u {delta, plural, =1{sekunda} other{# sekundi}}',
+    'in {delta, plural, =1{a year} other{# years}}' => 'u {delta, plural, =1{godina} other{# godina}}',
+    'in {delta, plural, =1{an hour} other{# hours}}' => 'in {delta, plural, =1{sat} other{# sati}}',
```

Comment:
Probably yes, but since yii2 official translation doesn't have  "few" and "many" I left that out. I don't know, maybe I should add it?